galpy/df/sphericaldf.py

```diff
@@ -81,8 +81,7 @@ def __init__(self,pot=None,denspot=None,rmin=None,rmax=None,
         self._denspot= self._pot if denspot is None else denspot
         if not conversion.physical_compatible(self._pot,self._denspot):
             raise RuntimeError("Unit-conversion parameters of input potential incompatible with those of the density potential")
-        self._rmin= 0. if rmin is None \
-            else conversion.parse_length(rmin,ro=self._ro)
+        conversion.parse_length(rmin,ro=self._ro)
```

Comment:
Should this be `rmin= conversion.parse_length(rmin,ro=self._ro)`?

Reply:
Oh, yep